### ISSUE TITLE
Some unicode characters are not stored in MAM for Postgres

### ISSUE THREAD

**Issue by ludwikbukowski** [esl/MongooseIM#1703]
MongooseIM version: b71e151c46
Installed from: source
Erlang/OTP version:  19

When MAM used with postgres with simple `mod_mam_meta` configuration:
```
 {mod_mam_meta, [
   {backend, odbc},
   {pm, [
     {user_prefs_store, mnesia}
   ]}
 ]}
```

**All messages containing special characters (like polish 'ś', 'ć') won't be stored in archive!**

The server error log when message "cześć" was sent:
```
(...)
2018-02-07 17:32:35.713 [error] <0.703.0> gen_server ejabberd_mod_mam_writer_localhost_2 terminated with reason: 
{{{case_clause,{{badarg,[{erlang,list_to_binary,
[[99,122,101,347,263]],[]},
{epgsql_binary,encode,3,[{file,"/Users/ludwikbukowski/work/mim/MongooseIM/_build/default/lib/epgsql/src/epgsql_binary.erl"
},{line,85}]},{epgsql_wire,encode_parameter,2,
[{file,"/Users/ludwikbukowski/work/mim/MongooseIM/_build/default/lib/epgsql/src/epgsql_wire.erl"},{line,221}]},{epgsql_wire,encode_parameters,5,
[{file,"/Users/ludwikbukowski/work/mim/Mongoo..."},...]},...]},...}},...},...} in gen_server:call/3 line 212
(...)
```

Looks like its the issue with `epgsql` library

**Comment by michalwski** [comment]
@ludwikbukowski @arcusfelis is it fixed in current master branch? There were some changes related to UTF handling in the database was it related to the issue?

**Comment by kzemek** [comment]
Fixed in #1748
